trie/verkle.go

```diff
@@ -163,14 +163,14 @@ func (t *VerkleTrie) UpdateAccount(addr common.Address, acc *types.StateAccount,
 		stem      = t.pointCache.GetTreeKeyBasicDataCached(addr[:])
 	)
 
-	binary.LittleEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
+	binary.BigEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
 	// get the lower 16 bytes of water and change its endianness
```

Comment:
not an issue with your PR, but what is this comment supposed to mean ?

Reply:
Ah, this is a left over when we had little endian. So it made sense with the deleted lines I guess? I'll remove it